src/stylesheets/components/BlogSection/BlogGrid.sass

```diff
@@ -1,7 +1,7 @@
 .blog-grid
-  @include flex
-  flex-wrap: wrap
+    @include flex
```

Comment:
I don't think using this mixin is a good idea. Here the output is:
```
align-items: center;
display: flex;
flex-direction: row;
justify-content: center;
flex-wrap: wrap;
```
While what you actually need is:
```
display: flex;
flex-direction: column;
```
Let's have a short pair programming and I'll show you what I mean.

Reply:
ok